<launch>
  <arg name="ns" default="scorpio/mmp0"/>

  <param name="caption_model_path" value="$(find caption_pkg)/src/log_st/model-best.pth" />
  <param name="infos_path" value="$(find caption_pkg)/src/log_st/infos_st-best.pkl" />
  <param name="imagenet_weights_path" value="$(find caption_pkg)/src/resnet101.pth"/>
  <param name="image_caption_image_topic" value="/camera/rgb/image_raw"/>

  <node name="caption_pmc_server" pkg="caption_pkg" type="caption_pmc_server.py" output="screen"/>
  <node name="caption_pmc_client" pkg="caption_pkg" type="caption_pmc_client.py" output="screen"/>

</launch>